archey/test/test_archey_output.py

```diff
@@ -271,38 +272,45 @@ def test_append_no_ansi_color(self, _, __, ___):
     @patch.dict(
         'archey.output.LOGOS_DICT',
         {
-            Distributions.DEBIAN: """\
-{c[0]} {r[0]} {c[1]}
-{c[0]} {r[1]} {c[1]}
-{c[0]} {r[2]} {c[1]}
-{c[0]} {r[3]} {c[1]}
-{c[0]} {r[4]} {c[1]}
-{c[0]} {r[5]} {c[1]}
-{c[0]} {r[6]} {c[1]}
-{c[0]} {r[7]} {c[1]}
-{c[0]} {r[8]} {c[1]}
-{c[0]} {r[9]} {c[1]}
-{c[0]} {r[10]} {c[1]}
-{c[0]} {r[11]} {c[1]}
-{c[0]} {r[12]} {c[1]}
-{c[0]} {r[13]} {c[1]}
-{c[0]} {r[14]} {c[1]}
-{c[0]} {r[15]} {c[1]}
-{c[0]} {r[16]} {c[1]}
-{c[0]} {r[17]} {c[1]}\
-"""
+            Distributions.DEBIAN: [
+                ' ',
+                ' ',
+                ' ',
+                ' ',
+                ' ',
+                ' ',
+                ' ',
+                ' ',
+                ' ',
+                ' ',
+                ' ',
+                ' ',
+                ' ',
+                ' ',
+                ' ',
+                ' ',
+                ' ',
+                ' '
+            ]
+        }
+    )
+    @patch.dict(
+        'archey.constants.COLOR_DICT',
+        {
+            Distributions.DEBIAN: ['FAKE_COLOR']
         }
     )
     @patch(
         'archey.output.print',
         return_value=None,  # Let's nastily mute class' outputs.
         create=True
     )
-    def test_centered_output(self, _, __, ___, ____):
+    def test_centered_output(self, print_mock, _, __, ___):
```

Comment:
I've changed some things here, maybe it requires some additional updates now too.